cli/src/commands/account.rs

```diff
@@ -30,7 +30,7 @@ pub enum Account {
     New {
         /// Seed the RNG with a numeric value
         #[clap(short = 's', long)]
-        seed: Option<u64>,
+        seed: Option<BigInt>,
```

Comment:
I'm wondering whether this shouldn't be BigUint

Reply:
Good catch, this should be `num_bigint::BigUint`